packages/synthetic-chain/src/cli/cli.ts

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env tsx
```

Comment:
doesn't the bin entrypoint need a shebang ?

Reply:
Good catch. It wasn't necessary in https://github.com/Agoric/agoric-3-proposals/actions/runs/7976691041/job/21777846307?pr=111 but I suppose it was using the `main` instead of `bin`. I've added the shebang for broader compat.